src/EUTelPatternRecognition.cc

```diff
@@ -22,11 +22,6 @@ namespace eutelescope {
 std::vector<EUTelTrack>& EUTelPatternRecognition::getTracks(){
 	return	_finalTracks; 
 }
-void EUTelPatternRecognition::clearEveryRun(){
```

Comment:
@AlexanderMorton This function is bloody useless, if this functionality is needed, implement it correctly!